src/registry/partners/index.scss

```diff
@@ -0,0 +1,3 @@
+@use '@utrecht/components/form-fieldset';
```

Comment:
Do we use 2 separate files for the css rules because we use them only in specific context?Asking because I usually see one file per component.

Reply:
Eventually in the package all the CSS gets bundled into a single chunk, FYI